### PR TITLE
Updated CPU and GPU Node Lists, and General and Priority Partition Li…

### PR DESCRIPTION
**Pull request by wmjosiah**
…sts to reflect current state of the cluster (1U CPU Expansion and Power9 nodes, mostly)

### CODE REVIEW

File: sheets-to-md/output/gpu-nodes.md [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
+| power9-gpu[001-006]  | IBM PowerSystem AC922 (PowerNV 8335-GTH) | 2x Power9 (32 Cores 128 Threads)                    | 2x NVIDIA Tesla V100 (32GB VRAM)          | 256GiB  | <ul><li>power9-gpu</li><li>power9-gpu-preempt</li></ul>     | <ul><li>power9le</li><li>ppc64le</li><li>altivec</li></li>v100</li></ul>                                               |
+| power9-gpu[007-008]  | IBM PowerSystem AC922 (PowerNV 8335-GTH) | 2x Power9 (32 Cores 128 Threads)                    | 2x NVIDIA Tesla V100 (32GB VRAM)          | 256GiB  | <ul><li>power9-gpu-osg</li><li>power9-gpu-preempt</li></ul> | <ul><li>power9le</li><li>ppc64le</li><li>altivec</li></li>v100</li></ul>                                               |
+| power9-gpu[009-016]  | IBM PowerSystem AC922 (PowerNV 8335-GTH) | 2x Power9 (40 Cores 160 Threads)                    | 4x NVIDIA Tesla V100 (16GB VRAM)          | 512GiB  | <ul><li>power9-gpu</li><li>power9-gpu-preempt</li></ul>     | <ul><li>power9le</li><li>ppc64le</li><li>altivec</li></li>v100</li></ul>                                               |
```
Comment (georgiastuart) [inline sheets-to-md/output/gpu-nodes.md:19]:
```suggestion
| power9-gpu[001-006]  | IBM PowerSystem AC922 (PowerNV 8335-GTH) | 2x Power9 (32 Cores 128 Threads)                    | 2x NVIDIA Tesla V100 (32GB VRAM)          | 256GiB  | <ul><li>power9-gpu</li><li>power9-gpu-preempt</li></ul>     | <ul><li>power9le</li><li>ppc64le</li><li>altivec</li><li>v100</li></ul>                                               |
| power9-gpu[007-008]  | IBM PowerSystem AC922 (PowerNV 8335-GTH) | 2x Power9 (32 Cores 128 Threads)                    | 2x NVIDIA Tesla V100 (32GB VRAM)          | 256GiB  | <ul><li>power9-gpu-osg</li><li>power9-gpu-preempt</li></ul> | <ul><li>power9le</li><li>ppc64le</li><li>altivec</li><li>v100</li></ul>                                               |
| power9-gpu[009-016]  | IBM PowerSystem AC922 (PowerNV 8335-GTH) | 2x Power9 (40 Cores 160 Threads)                    | 4x NVIDIA Tesla V100 (16GB VRAM)          | 512GiB  | <ul><li>power9-gpu</li><li>power9-gpu-preempt</li></ul>     | <ul><li>power9le</li><li>ppc64le</li><li>altivec</li><li>v100</li></ul>                                               |
```
`v100` was surrounded by `</li>` on both sides so it wasn't formatting the list correctly.

Comment (wmjosiah) [reply sheets-to-md/output/gpu-nodes.md:19]: Ack! Thank you!

Comment (georgiastuart) [reply sheets-to-md/output/gpu-nodes.md:19]: @wmjosiah not sure if this was autogenerated? May be an issue with the script?

**Review by georgiastuart — APPROVED** [review]
Looks good to me and builds correctly!

File: sheets-to-md/output/gpu-nodes.md (hunk 1 above)
Comment (wmjosiah) [reply sheets-to-md/output/gpu-nodes.md]: No just my mistake. Thanks so much. I missed in output too :)

**Pull request merged**